src/Resources/AbstractResource.ts

```diff
@@ -28,7 +28,7 @@ export default class AbstractResource {
         const api = this.api;
         const spec = new Spec(this.resourcePath, partialSpec);
 
-        return function makeResourceRequest(...args: Array<string | Function>): void {
+        return function makeResourceRequest(...args: Array<string | object | Function>): void {
```

Comment:
Ah gad dammit - I tried adding something here yesterday but not within the array! 🤦‍♀